### PR TITLE
Spelling: replaces colour by color

### PR DESCRIPTION
**Pull request by sciunto**
## Description

The more consistent we are, the less we do typos...

### CODE REVIEW

File: skimage/io/_plugins/_colormixer.pyx [hunk 1]
```diff
@@ -3,7 +3,7 @@
 #cython: nonecheck=False
 #cython: wraparound=False
 
-"""Colour Mixer
+"""color Mixer
```
Comment (jni) [inline skimage/io/_plugins/_colormixer.pyx]: Color (capital C)

File: skimage/data/__init__.py [hunk 2]
```diff
@@ -70,7 +70,7 @@ def camera():
 
 
 def astronaut():
-    """Colour image of the astronaut Eileen Collins.
+    """color image of the astronaut Eileen Collins.
```
Comment (jni) [inline skimage/data/__init__.py]: Color (capital C)

**Comment by soupault** [comment]
Does this PR have something to do with Brexit? :)

**Comment by sciunto** [comment]
Fixed.

**Pull request merged**